ops/helmfiles/dagster/migrate.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Migrates our dagster cluster DB schema
+# (taken from https://docs.dagster.io/deployment/guides/kubernetes/how-to-migrate-your-instance)
+
+function msg() {
+  echo $1 >&2
+}
+
+function scale_down() {
+  kubectl -n dagster scale deploy $1 --replicas=0
+}
+
+function scale_up() {
+  kubectl -n dagster scale deploy $1 --replicas=$2
+}
+
+export HELM_DAGSTER_RELEASE_NAME="monster"
+
+# Get the names of Dagster's Dagit and Daemon Deployments created by Helm
+DAGIT_DEPLOYMENT_NAME=$(kubectl -n dagster get deploy \
+  --selector=component=dagit -o jsonpath="{.items[0].metadata.name}")
+export DAGIT_DEPLOYMENT_NAME
+DAEMON_DEPLOYMENT_NAME=$(kubectl -n dagster get deploy \
+  --selector=component=dagster-daemon -o jsonpath="{.items[0].metadata.name}")
```

Comment:
you can combine the `export` statement with the assignment statement

Reply:
Hmm, shellcheck [does not like that](https://github.com/koalaman/shellcheck/wiki/SC2155), I might be missing something with that rule though.